lib/desktop_screens_new/history_screen/history_desktop.dart

```diff
@@ -65,7 +65,6 @@ class _HistoryDesktopScreenState extends State<HistoryDesktopScreen> {
               filteredFiles = getDisplayFileData(provider.allFilesHistory);
             },
             successBuilder: (provider) {
-              filteredFiles = getDisplayFileData(provider.allFilesHistory);
```

Comment:
does history screen refreshes when new data is received ?

Reply:
Yes, it worked